_episodes/006-rust-1.39.0.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "What's New in Rust 1.39"
+date: 2019-11-27T12:00:00+0000
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e006-rust-1.39.0.mp3
+duration: "43:03" #audio length
+length: "30996753" #filesize in byte
+#reddit: TODO
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+Jon and Ben review the long-awaited changes in [Rust 1.39](https://blog.rust-lang.org/2019/11/07/Rust-1.39.0.html).
+
+Rustacean Station is a community project; get in touch with us if you'd like to be interviewed, propose a topic for an episode, or help create the podcast itself!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+* [@1:03] - [References to by-move bindings in match guards](https://blog.rust-lang.org/2019/11/07/Rust-1.39.0.html#references-to-by-move-bindings-in-match-guards)
+
```

Comment:
You'll need to remove the empty lines here. They make it so that this is many paragraphs with one list each with one item, rather than one list with many items.